accepted/2025/cve-schema/cve.json

```diff
@@ -0,0 +1,210 @@
+{
+  "date": "2024-07-09",
+  "cves": [
+    {
+      "id": "CVE-2024-30105",
+      "title": ".NET Denial of Service Vulnerability",
+      "severity": "Critical",
+      "cvss": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H/E:U/RL:O/RC:C",
```

Comment:
Is the plan to use this field name even if the value later switches to CVSS 4.0 (or some other) format?

Reply:
This feels similar to me to how container tooling uses the `digest` field - a digest is a string of the form `<digest_type>:<digest_value>`, and the type tells tooling how to interpret the value. As long as the various CVE-description formats follow a similar philosophy, maybe the name of this field could be more generic ("characteristics"?) and the values in the field would be self-describing.

Reply:
My suggestion would be to do something like:

```json
"cvss": {"version": "3.1", "uri": "cvss:3.1:..."}
```

Reply:
The current format (`CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H/E:U/RL:O/RC:C`) is provided to us by the security team. The suggesting offered by @kaniini above looks good make the version explicit.

Reply:
Resolved.